### PR TITLE
Minor fixes in seeg-ct.sh

### PR DESCRIPTION
**Pull request by sipv**

### CODE REVIEW

File: bin/seeg-ct.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@
 # parameters of image processing can be tuned while looking at the results.
 
 
-if [ -z "$SUBJECT" || -z "$CT" ]
+if [ -z "$SUBJECT" ] || [ -z "$CT" ]
```
Comment (maedoc) [inline bin/seeg-ct.sh]: maybe the original syntax requires double square brackets? this is fine anyway, if it's working

File: bin/seeg-ct.sh [hunk 2]
```diff
@@ -69,8 +69,8 @@ label_with_dilation("masked_CT.nii", "dilated_CT.nii", "labeled_CT.nii")
 EOF
 
 echo compare labels to implantation schema
-echo mrview T1.nii.gz -overlay.load=dilated_CT.nii
-echo $open $implantation_pptx
+echo mrview T1.nii.gz -overlay.load labeled_CT.nii -overlay.interpolation_off -overlay.colourmap 3
+echo open implantation_pptx
```
Comment (maedoc) [inline bin/seeg-ct.sh]: I was thinking that some environment variables would do

- on mac, `open=open`
- on linux, `open=xdg-open`
-on windows (like we care, but for fun), `open=''`

but it's true that there isn't much we can do here

**Comment by maedoc** [comment]
if you don't have any other commits to push here, I'll merge; just let me know

File: bin/seeg-ct.sh (hunk 2 above)
Comment (sipv) [inline bin/seeg-ct.sh]: This is what I was thinking was the intention, but it seemed unnecessary to me to force the user to set the variables just to print it back to him.

**Comment by sipv** [comment]
Nothing else, it works perfectly.

**Pull request merged**